data/en/test_radio_mandatory.json

```diff
@@ -0,0 +1,42 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "0",
+    "title": "Other input fields",
```

Comment:
I think we should copy the group title into the survey title for all of these as `Other input fields` is confusing. It would also mean it would be clearly displayed as the survey title when viewing the surveys.